src/anyio/_core/_sockets.py

```diff
@@ -679,25 +673,22 @@ def wait_readable(obj: HasFileno | int) -> Awaitable[None]:
     return get_async_backend().wait_readable(obj)
 
 
-def wait_writable(obj: HasFileno | int) -> Awaitable[None]:
+def wait_writable(obj: FileDescriptorLike) -> Awaitable[None]:
     """
     Wait until the given object can be written to.
 
-    This does **NOT** work on Windows when using the asyncio backend with a proactor
```

Comment:
Do you think it's worth mentioning the thread implementation details here?

Reply:
Done.